lib/common/options.c

```diff
@@ -947,13 +947,28 @@ add_desc(GString *s, const char *tag, const char *desc, const char *values,
     free(escaped_en);
 }
 
+/*!
+ * \internal
+ * \brief Format option metadata as an OCF-compliant XML string
+ *
+ * \param[in] name         Daemon name
+ * \param[in] desc_short   Short description of the daemon
+ * \param[in] desc_long    Long description of the daemon
+ * \param[in] option_list  Options whose metadata to format
+ * \param[in] len          Number of items in \p option_list
+ *
+ * \return A string containing OCF-compliant option metadata XML
```

Comment:
Technically it's more OCF-like than OCF-compliant

Reply:
Ah yes... I don't see a lot of difference, but `<actions>` appears to be required by the OCF rng file, and we're using more `type` values (e.g., "time") than the OCF rng file allows.